### PR TITLE
fix: Modify UI review comments

### PR DESCRIPTION
**Pull request by xuanlid**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution

1.全局弹窗（标题问题，关闭按钮样式，对齐问题，按钮间距，表单间距，标题与内容间距）

2.全局搜索（图标样式和间距，搜索激活态描边，提示文本，搜索组件间距）

3.组件列表间距调整

4.选块组件（页签）灰背景圆角

5.区块管理-区块列表（列表样式，底部筛选样式和底部背景）

6.按钮样式调整

7.状态管理/资源管理代码示例背景，字体调整

8.页面schema分割线和面板阴影调整

9.国际化页面样式调整

10.左侧菜单图标样式调整

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Release Notes

- **New Features**
  - Enhanced user interface for various components, including improved button designs and new form items in the Bridge Setting.
  - Added sorting options in the block management panel for better organization.
  - Introduced confirmation callbacks for plugin switching and improved user interaction feedback in several components.

- **Bug Fixes**
  - Improved error handling in several components, providing clearer messages during failures, particularly in version handling and block deletion.

- **Style**
  - Updated styles across multiple components for better visual consistency and user experience, including hover effects and padding adjustments.
  - Adjusted layout properties, such as label widths and padding, to improve form alignment and responsiveness.
  - Introduced new CSS classes for enhanced styling in various components.

- **Refactor**
  - Simplified message handling in components to streamline notifications and improve clarity.
  - Enhanced component structure for better readability and organization.

These updates enhance usability, improve visual consistency, and refine user interactions across the application.

### CODE REVIEW

File: packages/common/component/BlockDeployDialog.vue [hunk 1]
```diff
@@ -254,7 +252,15 @@ export default {
     line-height: 0;
   }
 }
+.schema-compare {
+  :deep(.tiny-button) {
```
Comment (wenmine) [inline packages/common/component/BlockDeployDialog.vue]: 尽量不要写deep，看一下是否有别的处理方式

File: packages/plugins/block/src/BlockGroupArrange.vue [hunk 2]
```diff
@@ -48,25 +48,23 @@ export default {
...
-    height: 20px;
-    color: var(--ti-lowcode-text-color);
+    width: 24px;
+    height: 24px;
+    color: var(--te-common-icon-secondary);
     font-size: 16px;
-    border-radius: 2px;
+    border-radius: 4px;
+    background-color: var(--te-common-bg-container);
+    border: 1px solid var(--te-common-bg-container);
```
Comment (wenmine) [inline packages/plugins/block/src/BlockGroupArrange.vue]: border一般都不用bg，用border的common，找一下有没有对应的，如果没有则找设计师对齐

File: packages/plugins/block/src/BlockGroupArrange.vue [hunk 3]
```diff
@@ -48,25 +48,23 @@ export default {
...
     align-items: center;
     transition: 0.3s;
-    &:hover {
-      color: var(--ti-lowcode-common-primary-color);
-    }
     &.active {
-      color: var(--ti-lowcode-common-primary-color);
+      border: 1px solid var(--te-common-border-active);
+      color: var(--te-common-text-primary);
+      background-color: var(--te-common-bg-default);
```
Comment (wenmine) [inline packages/plugins/block/src/BlockGroupArrange.vue:66]: 感觉这里样式是不是和上面写反了，跟56行比

File: packages/plugins/block/src/Main.vue [hunk 4]
```diff
@@ -488,9 +488,23 @@ export default {
   right: 0;
   padding: 10px 16px;
   background-color: var(--ti-lowcode-component-search-bg);
+  box-shadow: 0 -2px 12px 0 rgba(0, 0, 0, 0.16);
```
Comment (wenmine) [inline packages/plugins/block/src/Main.vue]: box-shadow可以用base里面的box-shadow变量

File: packages/plugins/schema/src/Main.vue [hunk 5]
```diff
@@ -141,20 +141,22 @@ export default {
...
   background: var(--ti-lowcode-common-component-bg);
-  box-shadow: 2px 2px 6px rgb(0 0 0 / 60%);
+  box-shadow: 6px 0px 3px 0px rgba(0, 0, 0, 0.05);
   z-index: 1000;
   .source-code-header {
     display: flex;
     justify-content: space-between;
-    border-bottom: 1px solid var(--ti-lowcode-toolbar-border-color);
+    align-items: center;
+    border-bottom: 1px solid var(--ti-lowcode-base-split-color-1);
```
Comment (wenmine) [inline packages/plugins/schema/src/Main.vue]: 这里看一下是不是可以用 --te-common-border-divider分割线

File: packages/theme/base/src/component-common.less [hunk 6]
```diff
@@ -687,14 +766,22 @@
             &.col__center {
               text-align: left;
             }
+            .tiny-grid-required-icon {
+              align-self: center;
+              margin-top: 5px;
+            }
+            .tiny-grid-resizable.is__line:before,
+            .tiny-grid-thead-partition.is__line:before {
+              background-color: var(--te-common-border-hover);
```
Comment (wenmine) [inline packages/theme/base/src/component-common.less]: backgroud用border的场景吗

File: packages/plugins/block/src/BlockGroupArrange.vue (hunk 3 above)
Comment (xuanlid) [reply packages/plugins/block/src/BlockGroupArrange.vue:66]: 这个和别歌对过了颜色是正确的

Comment (xuanlid) [reply packages/plugins/block/src/BlockGroupArrange.vue:66]: 这个和UI对过是正确的颜色

File: packages/plugins/block/src/BlockSetting.vue [hunk 7]
```diff
@@ -209,7 +208,6 @@ export default {
           }
         }
         const title = '保存区块'
-        const status = 'custom'
```
Comment (hexqi) [inline packages/plugins/block/src/BlockSetting.vue:212]: status这个变量已经删除了，confirm里面的status参数不建议写成简写形式，可以去掉status参数或者直接写成status: ''

**Review by hexqi — DISMISSED** [review]

**Review by hexqi — APPROVED** [review]

**Pull request merged**